docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst

```diff
@@ -55,175 +58,154 @@ to start the daemon.
-``code`` object:
+   ``load_code`` returns an object of type ``Code``, which is the general AiiDA class for describing simulation codes. 
+
+Let's build the inputs for a new ``PwCalculation`` (defined by the ``quantumespresso.pw`` plugin, the default plugin for the code you chose before)
 
 .. code:: python
 
     builder = code.get_builder()
 
-This returns a builder that helps in setting up the inputs for the
-``PwCalculation`` class (associated to the ``quantumespresso.pw``
-plugin, the default plugin for the code you chose before).
-
-As the first step, you can assign a (short) label or a (long)
-description to the calculation that you are going to create, that you
-might find convenient in the future. This can be achieved with:
+As the first step, assign a (short) label or a (long) description to your
+calculation, that you might find convenient in the future.
 
 .. code:: python
 
     builder.metadata.label = "PW test"
     builder.metadata.description = "My first AiiDA calc with Quantum ESPRESSO on Si"
 
-This information will be saved in the database for later query or
+This information will be saved in the database for later queries or
 inspection. Note that you can press TAB after writing ``builder.`` to
-see all available inputs.
+see all inputs available for this calculation.
 
-Now you have to specify the number of machines (a.k.a. cluster nodes)
+Now, specify the number of machines (a.k.a. cluster nodes)
 you are going to run on and the maximum time allowed for the
-calculation. These general calculation options, that are independent of
-the code or plugin, but rather mainly passed later to the scheduler that
-handles the queue, are all grouped under “builder.options”:
+calculation. 
+The general options grouped under ``builder.options`` are independent of
+the code or plugin, and will be passed to the scheduler that handles the
+queue on your compute resource .
 
 .. code:: python
 
     builder.metadata.options.resources = {'num_machines': 1}
     builder.metadata.options.max_wallclock_seconds = 30 * 60
 
-Just like the normal inputs, these builder options are also
-TAB-completed. Type ``builder.metadata.options.`` and hit the TAB button
-to see the list of available options.
+Again, to see the list of available options, type
+``builder.metadata.options.`` and hit the TAB button.
 
 Preparation of inputs
 ~~~~~~~~~~~~~~~~~~~~~
 
-Quantum ESPRESSO requires an input file containing Fortran namelists and
-variables, plus some cards sections (the documentation is available
-`here <https://www.quantum-espresso.org/Doc/INPUT_PW.html>`_).
-The Quantum ESPRESSO plugin of AiiDA requires quite a few nodes in
-input, which are also documented
-`online <https://aiida-quantumespresso.readthedocs.io/en/stable/user_guide/calculation_plugins/pw.html>`_.
-Here we will instruct our calculation with a minimal configuration for
-computing the energy of silicon. We need:
+A Quantum Espresso calculation needs a number of input files:
```

Comment:
I would replace `input files` with `inputs`, because below you are not enumerating only files. Or combine point 2, 3 and 4 into one and say: an input file with a definition of a structure, calculation parameters and a set of kpoints.